cmd/api_key/api_key.go

```diff
@@ -148,6 +147,15 @@ var createApiKeyCmd = &cobra.Command{
 			if err != nil {
 				logrus.Fatal(err)
 			}
+
+			if authApi.RoleContainsSensitivePermissions(roleId) {
+				viper.BindPFlag("force", cmd.Flags().Lookup("force"))
+				err := util.ConfirmCommand(fmt.Sprintf("Some of the permissions of the role '%s' have security implications (such as user, API key, and role management operations). Are you sure you want to proceed with creating API Key '%s' with this role?", roleName, name), viper.GetBool("force"))
```

Comment:
you seems repeating `Some of the permissions of the role '%s' have security implications (such as user, API key, and role management operations).` quite a few time would be better to put this as const or var some where to avoid mistake and repeating.